docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3'
 services:
   siglens:
-    image: siglens/siglens:${SIGLENS_VERSION}
+    image: ${IMAGE_NAME}
```

Comment:
I think the `SIGLENS_VERSION` is never used since you're deleting it here, so you should delete it from the `environment` section below (and probably add `IMAGE_NAME` to that sections; but I'm not actually sure why we have those definitions in the `environment` section)

Reply:
I think it is not used docker-compose but it is used the install script